packages/backend/src/config/app.ts

```diff
@@ -123,17 +129,47 @@ function parseRedisTlsConfig(
 
   // self-signed certs.
   if (caFile !== '') {
-    options.ca = readFileSync(caFile)
+    options.ca = fs.readFileSync(caFile)
     options.rejectUnauthorized = false
   }
 
   if (certFile !== '') {
-    options.cert = readFileSync(certFile)
+    options.cert = fs.readFileSync(certFile)
   }
 
   if (keyFile !== '') {
-    options.key = readFileSync(keyFile)
+    options.key = fs.readFileSync(keyFile)
   }
 
   return Object.keys(options).length > 0 ? options : undefined
 }
+
+// exported for testing
+export function parseOrProvisionKey(keyPath: string): {
+  key: crypto.KeyObject
+  jwk: crypto.JsonWebKey
+} {
+  try {
+    const key = crypto.createPrivateKey(fs.readFileSync(keyPath))
+    const jwk = key.export({ format: 'jwk' })
+    if (jwk.crv === 'Ed25519') {
+      return { key, jwk }
```

Comment:
Given that the `jwk` isn't actually being included in the config and I think`parseOrProvisionKey` doens't actually need to generate the `jwk`, I vote we drop it.
(I'm trying to see how we'd generate an ed25519 private `KeyObject` from just `randomBytes` below)

```suggestion
export function parseOrProvisionKey(keyPath: string): crypto.KeyObject {
  try {
    const key = crypto.createPrivateKey(fs.readFileSync(keyPath))
    if (key.asymmetricKeyType === 'ed25519') {
      return key
```